src/pages/OfferNew/Checkout/Sign.tsx

```diff
@@ -127,16 +128,24 @@ export const Sign: React.FC<Props> = ({
       >
         <SignStatus signStatus={signStatus} />
       </motion.div>
-      <Disclaimer>
-        <MarkdownTranslation
-          textKey="CHECKOUT_SIGN_DISCLAIMER"
-          replacements={{
-            PREBUY_LINK: textKeys[getPrebuyPDFTextKey(insuranceType)](),
-            TERMS_LINK: textKeys[getInsurancePDFTextKey(insuranceType)](),
-          }}
-          markdownProps={{ linkTarget: '_blank' }}
-        />
-      </Disclaimer>
+      {offerData.quotes.map((quote) => {
+        return (
+          <Disclaimer>
```

Comment:
Same here, we could add a key but this is purely to break the design since the information is not valid if we don't fix the design